code/controllers/subsystem/mapping.dm

```diff
@@ -396,6 +407,9 @@ GLOBAL_LIST_EMPTY(the_station_areas)
 		if (VM.config_max_users > 0 && players > VM.config_max_users)
 			mapvotes.Remove(map)
 			continue
+		if(previous_maps && previous_maps[VM.map_name] > 7)
```

Comment:
Another magic value that needs to be a config

Reply:
![image](https://github.com/yogstation13/Yogstation/assets/5091394/13f8680d-30bd-47cc-8d21-82743b800181)